src/odemis/gui/comp/combo.py

```diff
@@ -223,7 +223,7 @@ def OnDrawItem(self, dc, rect, item, flags):
         else:
             # for painting the items in the popup
             # Draw color map
-            colorbar_width = r.width * COLOBAR_WITH_RATIO
+            colorbar_width = int(round(r.width * COLOBAR_WITH_RATIO))
```

Comment:
if width/height is checked to be bigger than 0 why do you need "round" here? int does the same isn't it?

Reply:
@Kleijwegt "int" always rounds down because it cuts off the decimals, but "round" rounds both up and down depending on the decimal. Only "int" returns an integer, "round" returns a floating point number.